### PR TITLE
XF-742 | update ember to 3.4.1 and CLI to 3.4.3

### PR DESCRIPTION
**Pull request by Bartosz-L**
Updated ember to 3.4.1.
But waiting for the release regarding the issue with head.hbs file
Link here: https://github.com/ember-cli/ember-cli/issues/8042

### CODE REVIEW

**Comment by Bartosz-L** [comment]
Bug with head.hbs fixed with newest cli update. Yay!

File: app/templates/components/autologin-frame.hbs [hunk 1]
```diff
@@ -1,3 +1,7 @@
 {{#if cookieSyncEnabled}}
-		<iframe style="display: none;" src="{{servicesDomain}}/autologin/frame"></iframe>
+		<iframe
+				style="display: none;"
```
Comment (xkxd) [inline app/templates/components/autologin-frame.hbs]: indentation is wrong here comparing to the others

File: package.json [hunk 2]
```diff
@@ -107,7 +107,7 @@
     "ember-responds-to": "1.5.0",
     "ember-router-scroll": "1.0.0",
     "ember-sinon": "2.2.0",
-    "ember-source": "3.3.2",
+    "ember-source": "~3.4.0",
```
Comment (xkxd) [inline package.json]: all versions are locked, let's lock these as well.

File: app/templates/components/autologin-frame.hbs [hunk 3]
```diff
@@ -1,3 +1,3 @@
 {{#if cookieSyncEnabled}}
-		<iframe style="display: none;" src="{{servicesDomain}}/autologin/frame"></iframe>
+		<iframe style="display: none;"src="{{servicesDomain}}/autologin/frame"></iframe>
```
Comment (hakubo) [inline app/templates/components/autologin-frame.hbs]: spaces

**Review by hakubo — APPROVED** [review]

**Pull request merged**